scripts/nuke.py

```diff
@@ -19,6 +20,7 @@
                         archive path for a job to kill and nuke
   --stale               attempt to find and nuke 'stale' machines
                         (e.g. locked by jobs that are no longer running)
+  --dry-run             Don't actually nuke anything
```

Comment:
So what does --dry-run actually do then except make it a noop.  Does it have any output? I guess, what's the use case here?